examples/recsys2020.ipynb

```diff
@@ -853,51 +1121,26 @@
     "We drop the columns, which are not required for training."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 22,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "DoneAndNotDoneFutures(done=set(), not_done=set())"
-      ]
-     },
-     "execution_count": 22,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "RMV = [x for x in train.columns if not(x in features+label_names+other)]\n",
-    "\n",
-    "train = train.drop(RMV,axis=1)\n",
-    "RMV = [c for c in RMV if c in valid.columns]\n",
-    "valid = valid.drop(RMV,axis=1)    \n",
-    "wait(train)\n",
-    "wait(valid)"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Our experiments show that we require only 10% of the training dataset. Our feature engineering, such as TargetEncoding, uses the training datasets and leverage the information of the full dataset. The actual model training does not require all the data, anymore."
+    "Our experiments show that we require only 10% of the training dataset. Our feature engineering, such as TargetEncoding, uses the training datasets and leverage the information of the full dataset. The actual model training does not require all the data, anymore.<br><br>\n",
+    "We sample the training dataset to 10% of the size and drop all columns, which we do not want to use."
```

Comment:
Why are we sampling down to 10% ? Is this to run faster or to save memory usage?

Reply:
Yes - during the competition we tried out 10%, 20% and 50% and we did not see an improvement in the performance by using more data. The reason is, that all the feature engineering (like Target Encoding) is done using the full dataset. Training the model on the features does not require all data anymore